src/simulation.c

```diff
@@ -154,7 +156,7 @@ void run_for(Simulation *sp, int step_cutoff) {
   }
 }
 
-void simulation_history_to_file(Simulation *sp) {
+void history_to_file(Simulation *sp) {
```

Comment:
In C, longer names are better. I would revert this back to `simulation_history_to_file`